README.adoc

```diff
@@ -276,21 +270,18 @@ To enable the address sanitizer set the RUNTIME_CHECK build flag to "asan":
 cmake .. -DCMAKE_C_FLAGS=-DQD_MEMORY_DEBUG -DRUNTIME_CHECK=asan
 ----
 
-On Aarch64, a hardware-assisted address sanitizer is enabled with "hwasan":
+On Aarch64, a hardware-assisted address sanitizer is enabled with `hwasan`.
+
+The ASAN (libasan) and UBSAN (libubsan) libraries must be installed in order to use this option.
 
 [source,shell script]
 ----
 cmake .. -DCMAKE_C_FLAGS=-DQD_MEMORY_DEBUG -DRUNTIME_CHECK=hwasan
 ----
 
-The ASAN (libasan) and UBSAN (libubsan) libraries must be installed in
-order to use this option.
-
-NOTE: Memory pool will produce false leak reports unless `QD_MEMORY_DEBUG`
-is also defined.
+NOTE: The memory pool produces false leak reports unless `QD_MEMORY_DEBUG` is also defined.
```

Comment:
`QD_MEMORY_DEBUG` now defines itself, @kgiusti we could safely delete this line, if we wanted. Maybe best to leave it around; people will not read this anyways (as witnessed by the number of people (me included) who struggled when this was not defined automatically).

Reply:
lets not touch this, it is not important for the topic of the PR